content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,130 @@
 ---
 title: "Getting Started"
-description: "Explore ZKsync-ethers SDK version 6"
-tags: ["zksync", "sdks", "ZKsync Era SDK", "ethereum", "era network", "javascript", "v6"]
+description: "Explore zksync-ethers SDK version 6"
+tags: ["zksync", "sdks", "zksync Era SDK", "ethereum", "era network", "javascript", "v6"]
 ---
 
-While most of the existing SDKs should work out of the box, deploying smart contracts or using unique ZKsync features,
-like account abstraction, requires providing additional fields to those that Ethereum transactions have by default.
+This document provides an overview of the zksync-ethers SDK for interacting with ZKsync Era.
 
-To begin, it is useful to have a basic understanding of the types of objects available and what they are
-responsible for, at a high level:
+## Key Concepts
 
-- `Provider` provides connection to the ZKsync Era blockchain, which allows querying the blockchain state,
-  such as account, block or transaction details, querying event logs or evaluating read-only code using call.
-  Additionally, the client facilitates writing to the blockchain by sending transactions.
-- `Wallet` wraps all operations that interact with an account. An account generally has a private key, which
-  can be used to sign a variety of types of payloads. It provides easy usage of the most common features.
+- **Provider**: Provides a connection to the ZKsync Era blockchain, enabling querying of blockchain state (e.g.,
+accounts, blocks, transactions) and sending transactions.
+- **Wallet**: Manages operations involving an account, typically using a private key for signing actions.
 
-## Examples
+## Installation
 
-The following examples could be easily run by writing the code snippets in a file and executing them as follows:
+Install TypeScript and the `zksync-ethers` library:
 
-```shell
-npx ts-node src/zksync.ts
+```bash
+npm install --save-dev typescript
```

Comment:
Let's remove the typescript install part. All they should need is the `npm install zksync-ethers ethers` step. 
Typescript can be installed globally or locally and depending on the user's setup you could cause issues making them install it locally. It is also already defined as a devDependency for the package.